documentation/modules/security/con-securing-client-authentication.adoc

```diff
@@ -51,20 +46,19 @@ spec:
   # ...
 ----
 
+The authentication type must match the equivalent configuration for the `Kafka` listener used to access the Kafka cluster.
```

Comment:
The authorization is cluster wide, not per-listener.

```suggestion
The authentication type must match the equivalent configuration in the `Kafka` custom resource.
```

Reply:
@scholzj This is authentication here, not authorization. 
That's per-listener, right?

I've added a line to _User authorization_ though:

> Authorization is cluster-wide.
> The authorization type must match the equivalent configuration in the `Kafka` custom resource.